google-cloud-trace/README.md

```diff
@@ -1,4 +1,5 @@
-# google-cloud-trace
+### <div align="center"> This library is not currently actively maintained.  We recommend that customers use [OpenTelemetry](https://opentelemetry.io/docs/languages/ruby/) for Cloud Trace instrumentation instead.
```

Comment:
Thanks! Can you please not include the div tags? (They won't work well with markdown.) And better to put this notice immediately below the "google-cloud-trace" heading below.

Reply:
Done.